### PR TITLE
Refactored Workorder Functionality To Modules

### PR DESCRIPTION
**Pull request by nialldonnellyfh**
# Motivation

This PR moves the Workorder functionality from the application to the fh-wfm-workorder and fh-wfm-workorder-angular modules.

Related to https://github.com/feedhenry-raincatcher/raincatcher-workorder/pull/15 and https://github.com/feedhenry-raincatcher/raincatcher-workorder-angular/pull/1

### CODE REVIEW

**Comment by witmicko** [comment]
LGTM

File: src/app/file/file.js [hunk 1]
```diff
@@ -11,35 +11,43 @@ angular.module('wfm-mobile.file', [
...
+            resolve: {
+              profileData: function(userClient) {
+                return userClient.getProfile();
+              },
+              files: function(fileClient, profileData) {
+                return fileClient.list(profileData.id);
+              },
+              file: function($stateParams, files) {
+                var yoke =  files.filter(function(file) {
+                  console.log('>>>>> resolve file');
```
Comment (vchepeli) [inline src/app/file/file.js]: better to add some console.log catcher as grunt task

**Comment by vchepeli** [comment]
LGTM

File: src/app/file/file.js (hunk 1 above)
Comment (witmicko) [inline src/app/file/file.js]: we need a proper logger and make eslint console.log an error.
you pointed this one out before as it was WIP, but got merged recently

**Pull request merged**